microprofile/WFLY-10712_microprofile_metrics.asciidoc

```diff
@@ -0,0 +1,230 @@
+= [WFLY-10712] Support Eclipse MicroProfile Metrics
+:author:            Jeff Mesnil
+:email:             jmesnil@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+:keywords:          observability,microprofile,metrics,openshift
+
+== Overview
+
+The goal of this feature is to increase observability of WildFly by exposing its metrics to monitoring tools such as https://prometheus.io[Promotheus]
+and the https://docs.openshift.com/container-platform/3.11/architecture/infrastructure_components/web_console.html[OpenShift Console].
+
+This feature adds support for the https://github.com/eclipse/microprofile-metrics/releases/download/1.1.1/metrics_spec-1.1.1.pdf[Eclipse MicroProfile Metrics 1.1.1 specification].
```

Comment:
Any idea when 1.2 will be available ? @pilhuhn

https://github.com/eclipse/microprofile-metrics/compare/1.1.1...master shows decent amount of commits

Reply:
We will not do a 1.2, but rather jump to 2.0 with many changes and also support for multiple applications per server. This is for the ~Feb 2019 release of MicroProfile.